src/utilities/ProvarDXUtility.ts

```diff
@@ -169,9 +169,7 @@ export default class ProvarDXUtility {
 
     private handleSpecialCharacters(password: string): string {
         if (password) {
-            password = password.split('&').join('"&"');
-            password = password.split('|').join('"|"');
-            password = password.split('^').join('"^"');
+            password = password.replace(/[\\^$.*+?()[\]{}|]/g, '\\$&');
```

Comment:
This doesn't look like the right fix to me. The example in the Jira is an invalid JSON string where the final character in the password is \ which is escaping the closing quote. That's just an error in the JSON not a bug in our code.

Reply:
I debugged it on my local and faced a similar issue where the commands were failing due to special characters in the password
**2X#$[3P8mj**
**&2Cp9%Wt4M**

So, that's why added a regex to escape special characters correctly.

Reply:
Now we have encoded the password value before sending it to Provar.